README.md

```diff
@@ -66,13 +70,17 @@ Symbol                              | NET35              | NET40              |
 
 * `FEATURE_APPDOMAIN` - enables support for features that make use of an AppDomain in the host.
 * `FEATURE_ASSEMBLYBUILDER_SAVE` - enabled support for saving the dynamically generated proxy assembly.
+* `FEATURE_COM` - enables support for COM Interop.
 * `FEATURE_EMIT_CUSTOMMODIFIERS` - enables emitting optional and required custom modifiers defined on parameters including return parameters. It seems like a defect in corefx not to expose these methods because they are still implemented.
 * `FEATURE_EVENTLOG` - provides a diagnostics logger using the Windows Event Log.
 * `FEATURE_GAC` - enables support for obtaining assemblies using an assembly long form name.
 * `FEATURE_GET_REFERENCED_ASSEMBLIES` - enables code that takes advantage of System.Reflection.Assembly.GetReferencedAssemblies().
 * `FEATURE_LEGACY_REFLECTION_API` - provides a shim for .NET 3.5/4.0 that emulates the `TypeInfo` API available in .NET 4.5+ and .NET Core.
+* `FEATURE_NETCORE_CONVERTER_API` - provides shims to implement missing Converter in .NET Core.

```

Comment:
Found it,  it is used in a test.  How about just defining a custom delegate for that?
https://github.com/castleproject/Core/blob/821047561ff957dac13e0e6d655395d0e0bae105/src/Castle.Core.Tests/GenClasses/GenClassWithGenMethods.cs#L43